src/DotLiquid/Util/StrFTime.cs

```diff
@@ -18,85 +20,197 @@ public static class StrFTime
             { "c", (dateTime) => dateTime.ToString("ddd MMM dd HH:mm:ss yyyy", CultureInfo.CurrentCulture) },
             { "C", (dateTime) => ((int)Math.Floor(Convert.ToDouble(dateTime.ToString("yyyy"))/100)).ToString() },
             { "d", (dateTime) => dateTime.ToString("dd", CultureInfo.CurrentCulture) },
+            { "D", (dateTime) => dateTime.ToString("MM/dd/yy", CultureInfo.CurrentCulture) },
             { "e", (dateTime) => dateTime.ToString("%d", CultureInfo.CurrentCulture).PadLeft(2, ' ') },
+            // E - not specified
+            // f - not specified
+            { "F", (dateTime) => dateTime.ToString("yyyy-MM-dd", CultureInfo.CurrentCulture) },
+            { "g", (dateTime) => dateTime.GetIso8601WeekOfYear("g") },
+            { "G", (dateTime) => dateTime.GetIso8601WeekOfYear("G") },
             { "h", (dateTime) => dateTime.ToString("MMM", CultureInfo.CurrentCulture) },
             { "H", (dateTime) => dateTime.ToString("HH", CultureInfo.CurrentCulture) },
+            // i - not specified
             { "I", (dateTime) => dateTime.ToString("hh", CultureInfo.CurrentCulture) },
             { "j", (dateTime) => dateTime.DayOfYear.ToString().PadLeft(3, '0') },
-            { "m", (dateTime) => dateTime.ToString("MM", CultureInfo.CurrentCulture) },
+            // J - not specified
             { "k", (dateTime) => dateTime.ToString("%H", CultureInfo.CurrentCulture) },
-            { "l", (dateTime) => dateTime.ToString("%h", CultureInfo.CurrentCulture) },
+            // K - not specified
+            { "l", (dateTime) => dateTime.ToString("%h", CultureInfo.CurrentCulture).PadLeft(2, ' ') },

```

Comment:
"l" format is changed. The change should be behind an opt-in flag (use the compatibility version pattern from microalps).
Same for other formats you adjusted.

Reply:
Keeping bug fixes behind an opt-in flag just causes frustrations. If you were using "Ruby" date then you were familiar enough to know it was wrong and used it anyways without reporting a bug. This is different than Liquid filters which are/were barely documented and were not bugs but simply deviations from the reference implementation (many times not so obvious).

Reply:
ok